### PR TITLE
Publish v3.9.10 and v3.10.2. [release]

### PR DESCRIPTION
**Pull request by JalexChen**

### CODE REVIEW

**Comment by JalexChen** [comment]
will need to wait for pyenv support

File: 3.10/Dockerfile [hunk 1]
```diff
@@ -32,7 +34,7 @@ RUN sudo apt-get update && sudo apt-get install -y \
 	curl https://pyenv.run | bash && \
 	sudo rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-RUN env PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install 3.10.1 && pyenv global 3.10.1
+RUN env PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install 3.10.2 && pyenv global 3.10.2
```
Comment (illia-v) [inline 3.10/Dockerfile:37]: Can we add this flag to `Dockerfile.template` along with upgrading the versions?

```suggestion
RUN env PYTHON_CONFIGURE_OPTS="--enable-shared --enable-optimizations" pyenv install 3.10.2 && pyenv global 3.10.2
```

See #108

**Comment by felicianotech** [comment]
Don't merge this yet please.

**Review by BytesGuy — APPROVED** [review]

File: 3.10/Dockerfile (hunk 1 above)
Comment (felicianotech) [reply 3.10/Dockerfile]: Thank you for the nudge. Your opened GitHub Issue has been completed and merged in, and we'll be adding the change to this PR.

**Comment by felicianotech** [comment]
Ready to go

File: 3.10/Dockerfile (hunk 1 above)
Comment (illia-v) [reply 3.10/Dockerfile]: Thank you! This should be a noticeable enhancement.

**Pull request merged**